scripts/cors/cors.json

```diff
@@ -0,0 +1,13 @@
+{
+  "CORSRules": [
+    {
+      "AllowedOrigins": [
+        "https://*.pulumi.com",
+        "https://*.pulumi-dev.io"
+      ],
+      "AllowedHeaders": ["Authorization"],
```

Comment:
Is this saying that auth headers are OK to send with these requests? If so, we probably should not be doing this -- but apologies if I'm reading this wrong.

Reply:
yes you are right. thanks christian, we should not be doing this.